docs/staking.md

```diff
@@ -0,0 +1,48 @@
+---
+id: staking
+title: Staking
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+HydraDX has a **long-running HDX staking program** which **incentivizes user activity** in areas that are beneficial to the Protocol. On this page you will find important information regarding the **mechanics behind the HDX Staking program**. You can also check out our [step-by-step guide on staking](/howto_stake).
+
+## Staking Basics {#basics}
+
+HDX holders can **stake their HDX and receive rewards** which become **claimable as time passes**. Staking rewards are distributed from a dedicated pot that is gradually filled up by **different Protocol revenue streams**. Initially, the main revenue stream are the **LP fees** which the HydraDX Protocol accrues from its **HDX LP position in the Omnipool**. Furthermore, there is a proposal to subsidise APR during the first year of the staking program by **allocating an additional subsidy of ~22M HDX** from the HydraDX Treasury.
```

Comment:
The proposal for subsidised rewards has already been approved, it will be scheduled when staking is initialised to start distributing 3600 blocks after staking starts - so you can probably just say that token holders approved the subsidy of 22M HDX for the first year